### PR TITLE
Travis CI Integration

### PR DESCRIPTION
**Pull request by jplyle**
http://jira.webinos.org/browse/WP-793

This pull request contains all the configuration needed to get the Webinos Project using Travis CI for testing.  It also contains a few minor edits to tests and to the PZP to enable better testing.

### CODE REVIEW

File: webinos_config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "ports"             :{
-    "provider"          :80,
-    "provider_webServer":443,
+    "provider"          :6080,
+    "provider_webServer":6443,
```
Comment (habibvirji) [inline webinos_config.json]: Any reason why ports are changed

**Comment by jplyle** [comment]
Wow, I've never submitted a pull request that needed rebasing quite so quickly...

File: webinos_config.json (hunk 1 above)
Comment (jplyle) [inline webinos_config.json]: That's been changed, now.

Comment (jplyle) [inline webinos_config.json]: $%£! Some kind of rebase errors have brought this back in.

**Pull request merged**